package.json

```diff
@@ -19,8 +20,8 @@
     "eslint": "^8.22.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-vue": "^9.3.0",
-    "prettier": "2.7.1",
     "node-sass": "^7.0.1",
+    "prettier": "2.7.1",
```

Comment:
This can be safely moved to devdependencies 😉

Reply:
Done 😸